protocol/xmrtaker/net_test.go

```diff
@@ -1,18 +1,21 @@
 package xmrtaker
 
 import (
+	"path"
 	"testing"
 
+	"github.com/noot/atomic-swap/common"
 	"github.com/noot/atomic-swap/common/types"
 
+	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/stretchr/testify/require"
 )
 
-func newTestXMRTaker(t *testing.T) *Instance {
-	b := newBackend(t)
+func newTestXMRTaker(t *testing.T, ec *ethclient.Client) *Instance {
```

Comment:
same here!

Reply:
See my reply on `newTestXMRMaker` above and let me know if you have an alternative solution.